src/analyzing/AST_to_IL.ml

```diff
@@ -112,6 +112,15 @@ let fixme_stmt kind gany =
 (*****************************************************************************)
 (* Helpers *)
 (*****************************************************************************)
+
```

Comment:
You need a changelog entry, `vim changelog.d/pa-3056.fixed` and write a user-oriented description of the change. The example that you used in the PR comment it's perfect IMO.

Reply:
Got it. This one was a bit fuzzy for me because it changes the CFG, but I wasn't sure if the change affects the behavior that the user sees or not. I suppose the user can always use the `cfg_il` flag and this is considered user-facing.

By the way,  I like it that you specify `vim`. 😃

Reply:
Added changelog.